versioned/interface.ts

```diff
@@ -50,7 +51,7 @@ abstract class Versioned {
   readonly version: version;
   readonly contractAddress: string;
 
-  public static FETCH_COUNT = 20;
+  public static FETCH_COUNT = 5;
```

Comment:
Maybe `10` would be better? I think 5 by 5 may be a bit slow

Reply:
I changed to `10`